src/System.Security.Cryptography.Xml/tests/RSAKeyValueTest.cs

```diff
@@ -96,7 +96,16 @@ public void LoadXml_InvalidXml(string xml)
             xmlDocument.LoadXml(xml);
 
             RSAKeyValue rsa = new RSAKeyValue();
-            Assert.Throws<CryptographicException>(() => rsa.LoadXml(xmlDocument.DocumentElement));
+
+            // FormatException exception because desktop does not
+            // check if Convert.FromBase64String throws
+            // Related to: https://github.com/dotnet/corefx/issues/18690
```

Comment:
Actually we're on the same page, probably this was caused by code reviewing on my phone :) 

In my experience, product code that gathers bug numbers in it starts to decline in quality. The code gets harder to read, and future devs see a minefield and start maintaining it more .. tactically. (Maybe I'm heavily influenced by working in the depths of Visual Studio... and there the bug numbers lost all meaning after 15 years) 

Changeset descriptions are the perfect place for historical notes like bug numbers.

I'm not so bothered about tests. Tests are often records of the odd behavior of product code, and from time to time it makes sense to document why they existing using bug numbers.

That's just my take though -- I should probably try to gather consensus around some update to the coding guidelines.